va/va.h

```diff
@@ -4174,6 +4174,24 @@ VAStatus vaEndPicture(
     VAContextID context
 );
 
+/**
+ * Same behavior with vaEndPicture except a sync fd list for synchronizations.
+ * if sync_num = 0 or sync_fds == NULL, the behavior should be same with vaEndPicture.
+ * if sync_num >= 1, the sync_fds[0] should be the fence out, this fd(or the fence
+ * behind this fd)  will be overridden by this call, and will be signaled to indicate
+ * current frame finishing.
+ * sync_fds[1] to sync_fds[1 ~ sync_num-1] is fence in, current HW execution
+ * will be blocked until all these fences are signaled.
+ * these fence fds is file descriptor of dma_fence.
```

Comment:
Is it possible to call vaEndPicture2() with a fence-out but no fence-in(s)? If so it'd be good to specify it in this function comment (and maybe other expected example use patterns);